Core/ModuleInstaller.cs

```diff
@@ -967,7 +967,12 @@ public void AddRemove(ref HashSet<string> possibleConfigOnlyDirs, RegistryManage
         /// </summary>
         public void Upgrade(IEnumerable<string> identifiers, IDownloader netAsyncDownloader, ref HashSet<string> possibleConfigOnlyDirs, RegistryManager registry_manager, bool enforceConsistency = true)
         {
-            var resolver = new RelationshipResolver(identifiers.ToList(), null, RelationshipResolver.DependsOnlyOpts(), registry_manager.registry, ksp.VersionCriteria());
+            var resolver = new RelationshipResolver(
+                identifiers.ToList(),
+                identifiers.ToList(),
```

Comment:
This looks really weird, but after some thought it's clearly right. Can we add a `//` comment explaining what we're doing?

Also it might be nice to call `.ToList()` once and store it in a `var`.